R/setup.R

```diff
@@ -69,8 +69,11 @@ metadata_create_template <- function(dataset_id,
                            austraits_curators = "unknown",
                            assistants = "unknown"
                            ),
-       dataset = list(year_collected_start = "unknown",
-                      year_collected_end = "unknown",
+       dataset = list(custom_R_code = NA,
+                      collection_date = "unknown",
+                      taxon_name = NA,
+                      site_name = NA,
+                      context_name = NA,
```

Comment:
Yes, it is intentional, because the default for at least site_name and context_name is that they don't apply, since these aren't fields for all datasets. However, there is always a collection_date. It just might not be known